File: C Programming/Assignment/A3/factorial of number.c
#include<stdio.h>
void main()
{
	int num;
	int fact=1,i=1;
	printf("enter the number");
	scanf("%d",&num);
	while(i<=num)
	{
		fact=fact*i;
		i++;
	}
	printf("factorial of given  number is %d",fact);
	
}

File: Core java/CtoJava/src/assignment3/FactorialOfNumber.java
package assignment3;

public class FactorialOfNumber {
	public static void main(String[] args)
	{
		int num=5;
		int fact=1,i=1;
		while(i<=num)
		{
			fact=fact*i;
			i++;
		}
		System.out.printf("factorial of given  number is %d",fact);
		
	}

}
